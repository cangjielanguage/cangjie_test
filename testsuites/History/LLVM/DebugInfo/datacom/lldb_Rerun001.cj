// DEPENDENCE: ../cjcmd.py ../DatacomTest.py lldb_Rerun001_expected.txt ../basecase/022_debug_launch.cj
// EXEC: cjc -g 022_debug_launch.cj
// EXEC: python3 DatacomTest.py lldb_Rerun001_expected.txt



