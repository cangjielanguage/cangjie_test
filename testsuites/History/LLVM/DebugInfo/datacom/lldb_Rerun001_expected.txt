lldb,lldb 022_debug_launch,022_debug_launch
r,r,exited with status